Gemfile

```diff
@@ -29,10 +26,17 @@ end
 
 group :dev do
   gem "bump"
+  gem "yard"
+  gem "rubocop", "~> 0.65.0", require: false
 end
 
 group :dev, :test do
-  gem "byebug", platforms: :ruby
+  platform :ruby do
+    gem "pry"
+    gem "pry-nav"
+    gem "pry-stack_explorer"
+    gem "pry-doc"
+  end
```

Comment:
pry > byebug

Reply:
I disagree! 🙂 All good though, your choice 🙂

Reply:
Interesting. I'd be happy to dig deeper if you have the time, I might learn something.